tests/input/arw_raw/test_arw_raw.py

```diff
@@ -11,15 +11,22 @@
 
 from preview_generator.exception import UnavailablePreviewType
 from preview_generator.manager import PreviewManager
-from preview_generator.utils import executable_is_available
 from tests import test_utils
 
 CURRENT_DIR = os.path.dirname(os.path.abspath(__file__))
 CACHE_DIR = "/tmp/preview-generator-tests/cache"
 IMAGE_FILE_PATH = os.path.join(CURRENT_DIR, "DSC08523.ARW")
```

Comment:
The test image has a size of 23.8MB, wouldn't it be possible to have a smaller one? This would decrease the network usage to clone the repo and probably make this test quicker.

Reply:
I'm not sure it's possible easily to generate real raw image outside of a true camera, but i may tried to get a RAW file from a NEX5 (14mpx) instead of one from a A6000 (24,3mpx which should have better size.

Reply:
I change the image to a smaller one (14,7MB) from NEX5 camera.